v3.3/glfw/glfw/include/dummy.go

```diff
@@ -1,4 +1,3 @@
-//go:build required
```

Comment:
We probably shouldn't drop this line.

I think the fix is to update `grab-upstream.sh`—it currently writes just `// +build required`. The new `go:build` lines were probably added by gofmt in the past, but it's better to update the source `grab-upstream.sh` to include them to begin with.

All supported versions of Go long since understand `go:build`, so `+build` should be fine to drop. Up to you.

Reply:
I thought I had fixed them all but it seems not. Will update the script to add `go:build`.